derivation_method/super_method.py

```diff
@@ -330,7 +330,8 @@ def retrieve_actions(self):
                     {
                         "original_col": self.meta['from_class_short_label'],
                         "new_col": self.meta['to_class_short_label'],
-                        "term_pairs": self.meta['term_pairs']
+                        "term_pairs": self.meta['term_pairs'],
+                        "remove_unmapped_rows": True if self.meta["m"]["remove_unmapped_rows"]=='true' else False
```

Comment:
@PranjaliParnerkar  
getting an error when running a previously created Method without remove_unmapped_rows:
![image](https://github.com/GSK-Biostatistics/tab2neo/assets/42695162/a50880e8-bae8-4d79-8693-e05c15401367)
Please make it backwards compatible.